engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/UniverseSetupScreen.java

```diff
@@ -315,6 +330,19 @@ public void setEnvironment(UniverseWrapper wrapper) {
         }
     }
 
+    public int findIndex(List<WorldSetupWrapper> worlds,String worldName)
```

Comment:
You might want to add Javadoc to a place like this to explain the return value (that `-1` means the index value wasn't found)